src/generate_view.c

```diff
@@ -52,6 +64,22 @@ void generate_view(SDL_Window* p_window, map_t* p_map)
     SDL_Rect player_rect = {  p_map->player.current_pos.x, p_map->player.current_pos.y,  w, h};
     SDL_RenderCopy(p_renderer, p_player_texture, NULL, &player_rect);
 
+    SDL_QueryTexture(p_heart_texture, NULL, NULL, &w, &h);
+    SDL_Rect heart_rect = { p_map->space.x_max-50, p_map->space.y_max,  w, h };
+    SDL_RenderCopy(p_renderer, p_heart_texture, NULL, &heart_rect);
+
+    itoa(p_map->player.heart, heartNum, 10);   
```

Comment:
Unfortunately itoa is not an standard function! I don't have it in MAC check https://stackoverflow.com/questions/10162465/implicit-declaration-of-function-itoa-is-invalid-in-c99.

Can you please replace line 71 with this line:
sprintf(heartNum, "%d", p_map->player.heart);

it does exactly the same thing.

Reply:
ok